analysis_options.yaml

```diff
@@ -1 +1,40 @@
+# https://dart.dev/guides/language/analysis-options
 include: package:dart_flutter_team_lints/analysis_options.yaml
+
+analyzer:
+  language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
+
+linter:
+  rules:
+    - avoid_bool_literals_in_conditional_expressions
```

Comment:
I'm curious - where are you getting these rules from? See https://github.com/dart-lang/ecosystem/issues/110

Reply:
In this case, `http_parser`